apps/web/src/components/collectibles/collectible-browser.tsx

```diff
@@ -71,14 +71,22 @@ export default function CollectibleBrowser({
     <div className={css.root}>
       <Heading className={css.title}>{collectible.title}</Heading>
       <div className={css.imageWrapper}>
-        <Image
-          src={collectible.image}
-          loader={cmsImageLoader}
-          width={700}
-          height={700}
-          layout="responsive"
-          objectFit="contain"
-        />
+        { !collectible.previewVideo && 
```

Comment:
We might want to have a "show cover" link that flips between showing the video/image.